src/modals/ChangeNominations/index.tsx

```diff
@@ -37,10 +33,7 @@ export const ChangeNominations = () => {
   const controller = getBondedAccount(activeAccount);
   const signingAccount = isPool ? activeAccount : controller;
 
-  const nominations =
```

Comment:
Pool is disabling via feature flag. Why we need these changes?

Reply:
The codebase uses 
```js
const { membership } = usePoolMemberships();
``` 
which does not hide the property `isPool`. It behaves as if it always pools, so it fails with nominations logic. Therefore, I removed this logic. I believe that removing it is better than extracting this logic and creating a condition for it.